app/views/unattended/provisioning_templates/snippet/first_boot_setup.erb

```diff
@@ -0,0 +1,14 @@
+<%#
+kind: snippet
+name: first_boot_setup
+model: ProvisioningTemplate
+snippet: true
+description: |
+  Post replacement.
+-%>
+<%= save_to_file('/etc/systemd/system/first_boot_setup_service.service',
```

Comment:
Idea: Do we need two snippets, first_boot_setup & for non-systemd? This could be just one snippet with one condition, rendering two things

Reply:
Yes, that would be probably better.